letsencrypt-nginx/letsencrypt_nginx/tls_sni_01.py

```diff
@@ -65,8 +65,8 @@ def perform(self):
                 if addr.default:
                     addresses.append([obj.Addr.fromstring(default_addr)])
                     break
-            else:
-                addresses.append(list(vhost.addrs))
+                else:
```

Comment:
Nope, is not correct. Thank you for highlight it. 
I was thinking that this `else` was related with the `if` inside of the last for loop but is not, it's related with the if in the L:57, also this else is unnecessary because we're returning if not found a vhost.